### DISCUSSION TITLE
How do I find and edit my saved searches?

### DISCUSSION THREAD

**Discussion by geoff-maddock** [community/community#53150] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I performed a search on github, then clicked the "save" option that let me name the search for later usage.

Where can I find this search to apply it?   I can't find a static page with it listed, only when I click into the search bar and type the name of the search does it come up for me.

Where can I go back and edit this search?

**Comment by colinwm** [comment, marked as answer]
Good question @geoff-maddock, sorry this is not well documented. If you type `saved:` into the search bar (or just the first few characters of the saved search name) it'll be suggested as a completion, e.g. 

<img width="536" alt="image" src="https://user-images.githubusercontent.com/78169714/232919914-993c612c-8084-485b-b74a-a7aa295b9ac4.png">

Then you can begin typing more search terms which will further restrict the search within that scope.

**Comment by ginomiglio** [comment]
>Where can I go back and edit this search?

Does this feature exist?

**Reply by colinwm** [reply]
Yes (although it's a bit hard to find, sorry about that). Just type `saved:` into the search box, and then click on "manage saved searches", and you'll be able to edit them.

**Reply by ginomiglio** [reply]
my bad – I completely missed that in your original screenshot

**Comment by dsantiago** [comment]
It's still hard to find, a page to manage saved queries would be appreciated.

**Comment by muzimuzhi** [comment]
Related section in docs.github.com:
[Creating and managing saved searches](https://docs.github.com/en/search-github/github-code-search/using-github-code-search#creating-and-managing-saved-searches)

**Reply by Julian-Dumitrascu** [reply]
Worse than other software.

**Comment by zeroindex** [comment]
Ok, for me it took me a second to realize that you cannot have the repo in the search at all. The ONLY thing you type, as shown in the screenshot above, is `saved:`

I thought leaving the repo would show me the saved filters per repo. And also, GitHub will prefill it with a default repo too. So just delete that all and do exactly what you see in the screenshot.

**Comment by jeromedalbert** [comment]
It looks like the "Manage saved searches" option is not available any more. I have filed a bug [here](https://github.com/orgs/community/discussions/205549).

**Comment by anthonywhite** [comment]
my 2 cents - just use browser favourites...
